File: C  programs/Problems on Pattern Printing/String Pattern/Square  String Pattern.c
/* Write a program which accept String from user and display below  pattern.

Input : "Hello"
Output :  H  e  l  l  o
          H  e  l  l  o 
		  H  e  l  l  o
		  H  e  l  l  o         

Input : "Marvellous"
Output : M a r v e l l o u s
         M a r v e l l o u s
		 M a r v e l l o u s
		 M a r v e l l o u s 
		 M a r v e l l o u s
		 M a r v e l l o u s
		 M a r v e l l o u s
		 M a r v e l l o u s
		 M a r v e l l o u s
		 M a r v e l l o u s

*/

#include<stdio.h>

void Pattern(char str[])
{
	int i=0;
	int j=0;
	int k=0;
	int iCnt=0;
	int iLength=0;

	while(str[iLength]!='\0')
	{
		iLength++;
		iCnt++;
	}

	for(i=1;i<=iLength;i++)
	{
		k=0;

		for(j=1;j<=iLength;j++)
		{
			printf("%c\t",str[k]);
			k++;
		}
		printf("\n");
	}

}

int main()
{
	char str[30]={'\0'};

	printf("Enter string: \n");
	scanf("%[^'\n']s",&str);

	Pattern(str);

	return 0;
}

File: Java programs/Problems on Pattern Printing/String Pattern/Square string pattern.java
/* Write a java program which accept String from user and display below  pattern.

Input : Hello
Output :  H  e  l  l  o
          H  e  l  l  o 
		  H  e  l  l  o
		  H  e  l  l  o          

*/
		   
import java.util.*;

class Pattern
{
	public void Pattern(String str)
	{
		char Arr[]=str.toCharArray();
		int i=0,j=0;
		
		for(i=0;i<Arr.length;i++)
		{
			for(j=0;j<Arr.length;j++)
			{
				System.out.print(Arr[j]+"\t");
			}
            System.out.println();
		}
	}
}

class Lba381
{
    public static void main(String arg[])
	{
         Scanner sobj=new Scanner(System.in);
		 
		 System.out.println("Enter String");
		 String str=sobj.nextLine();
		 
		 Pattern pobj=new Pattern();
		 pobj.Pattern(str);
	}
}	
		